assets/css/_map_page.scss

```diff
@@ -41,7 +44,13 @@ $map-page-search-panel-width: 24rem;
 
   filter: drop-shadow(2px 0px 10px rgba(0, 0, 0, 0.2));
 
-  hr {
+  & > div {
+    max-height: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  & > div > hr {
```

Comment:
Hmm, I'm not seeing the `hr` as it's proper width anymore? It seems to just be a dot and this style isn't being applied, maybe remove the specificity?

![image](https://github.com/mbta/skate/assets/6913198/277f87eb-1ca2-4e8a-80df-bb984d329621)


```suggestion
  & > hr {
```

Reply:
Good catch - could have sworn I checked that that was working.